src/psyclone/tests/nemo/transformations/openacc/kernels_directive_test.py

```diff
@@ -56,20 +56,6 @@
                  "end program do_loop\n")
```

Comment:
Please check this ...

C: 44, 0: third party import "from psyclone.psyGen import PSyFactory, TransInfo, InternalError" should be placed before "from fparser.common.readfortran import FortranStringReader" (wrong-import-order)
C: 45, 0: third party import "from psyclone.transformations import TransformationError" should be placed before "from fparser.common.readfortran import FortranStringReader" (wrong-import-order)

Reply:
Again, my pylint is happy either way so I've changed it.